Makefile

```
@@ -161,7 +161,9 @@ check:
 
 .PHONY: generate
 generate:
-	@hack/generate.sh ./cmd/... ./extensions/... ./pkg/... ./plugin/... ./test/...
+	@hack/update-protobuf.sh
+	@hack/update-codegen.sh
+	@hack/generate-parallel.sh
```

Nit: can you move the package roots list from `hack/generate-parallel.sh` to here and pass it as an argument similar to the other make targets?
This way, when adding a new package root, one directly knows they should add it here, too.
```suggestion
	@hack/generate-parallel.sh cmd extensions pkg plugin test
```

---

Sure, I'll do it.

---

Done